pyproject.toml

```diff
@@ -77,7 +78,6 @@ dependencies = [
   "mkdocstrings~=0.22.0",
   "markdown-exec[ansi]~=1.6.0"
 ]
-detached = true
```

Comment:
I am curious, why is the "docs" environment no longer detached?

Reply:
The environment is no longer attached as an in-repo example of the `hatch-pip-compile` plugin. Once I have a solution for #8 up I will add some CLI documentation using [mkdocs-click](https://github.com/mkdocs/mkdocs-click) that will actually take advantage of being "attached".